browser-switch/build.gradle

```diff
@@ -77,76 +78,73 @@ artifacts {
     archives sourcesJar
 }
 
-signing {
-    required {
-        !version.endsWith("DEVELOPMENT")
-    }
-    sign configurations.archives
-}
+def sonatypeUsername = System.env['SONATYPE_USERNAME']
+def sonatypePassword = System.env['SONATYPE_PASSWORD']
 
 nexusStaging {
     packageGroup = 'com.braintreepayments'
+    stagingProfileId = 'd2424106a9f1f'
+    username = sonatypeUsername
+    password = sonatypePassword
 }
 
-def sonatypeUsername = System.env['SONATYPE_USERNAME']
-def sonatypePassword = System.env['SONATYPE_PASSWORD']
-
-uploadArchives {
-    repositories {
-        mavenDeployer {
-            beforeDeployment { MavenDeployment deployment -> signing.signPom(deployment) }
-
-            if (version.endsWith("DEVELOPMENT")) {
-                repository(url: 'file:///tmp/maven/')
-            } else {
-                repository(url: 'https://oss.sonatype.org/service/local/staging/deploy/maven2/') {
-                    authentication(userName: sonatypeUsername, password: sonatypePassword)
-                }
-                snapshotRepository(url: 'https://oss.sonatype.org/content/repositories/snapshots/') {
-                    authentication(userName: sonatypeUsername, password: sonatypePassword)
-                }
-            }
+signing {
+    required {
+        !version.endsWith("DEVELOPMENT")
+    }
+    sign publishing.publications
+}
 
-            pom.setArtifactId 'browser-switch'
+afterEvaluate {
+    publishing {
+        publications {
+            release(MavenPublication) {
+                // Applies the component for the release build variant.
+                from components.release
 
-            pom.project {
-                name 'browser-switch'
+                // You can then customize attributes of the publication as shown below.
                 groupId = 'com.braintreepayments'
+                artifactId = 'browser-switch'
                 version = "${android.defaultConfig.versionName}"
-                packaging 'aar'
-                description 'A library making it easy to open a url in a browser or Chrome Custom Tab and receive a response from the web page.'
 
-                url 'https://github.com/braintree/browser-switch-android'
+                pom {
+                    name = 'browser-switch'
+                    packaging = 'aar'
+                    description = 'A library making it easy to open a url in a browser or Chrome Custom Tab and receive a response from the web page.'
```

Comment:
Do we want to align this with the README?

> Android Browser Switch makes it easy to open a url in a browser or Chrome Custom Tab and receive a response as the result of user interaction, either cancel or response data from the web page.

Reply:
Yeah this is also a good call.